Mathlib/Combinatorics/SimpleGraph/Girth.lean

```diff
@@ -40,6 +40,15 @@ lemma exists_girth_eq_length :
       ← exists_prop, Subtype.exists', Sigma.exists', eq_comm] at h ⊢
     exact ciInf_mem _
 
+lemma three_le_length : 3 ≤ G.girth := by
+  by_cases h : G.IsAcyclic
+  · rw [← girth_eq_top] at h
+    rw [h]
+    apply le_top
+  · rw [← exists_girth_eq_length] at h
+    have ⟨_, ⟨_, _⟩⟩ := h
```

Comment:
Pointy brackets automatically right-associate (well, the rule is more complicated, but you can remember it like that)
```suggestion
    have ⟨_, _, _⟩ := h
```